nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -1042,9 +1042,9 @@ New and extended interfaces:
 - `lib.fetchers.normalizeHash`: Convert an attrset containing one of `hash`, `sha256` or `sha512` into one containing `outputHash{,Algo}` as accepted by `mkDerivation`.
 - `lib.fetchers.withNormalizedHash`: Wraps a function which accepts `outputHash{,Algo}` into one which accepts `hash`, `sha256` or `sha512`.
 - Various builtins are now reexported in a more standard way:
-  - `lib.map` -> `lib.lists.map` -> `builtins.map`
+  - `lib.map` -> `lib.lists.map` -> `map`
   - `lib.intersectAttrs` -> `lib.attrsets.intersectAttrs` -> `builtins.intersectAttrs`
-  - `lib.removeAttrs` -> `lib.attrsets.removeAttrs` -> `builtins.removeAttrs`
+  - `lib.removeAttrs` -> `lib.attrsets.removeAttrs` -> `removeAttrs`
```

Comment:
Should we revert those changes or extend them:

```diff
-  - `lib.map` -> `lib.lists.map` -> `builtins.map`
+  - `lib.map` -> `lib.lists.map` -> `builtins.map` -> `map`
   - `lib.intersectAttrs` -> `lib.attrsets.intersectAttrs` -> `builtins.intersectAttrs`
-  - `lib.removeAttrs` -> `lib.attrsets.removeAttrs` -> `builtins.removeAttrs`
+  - `lib.removeAttrs` -> `lib.attrsets.removeAttrs` -> `builtins.removeAttrs` -> `removeAttrs`
```

Reply:
I don't think we should be editing the 24.11 release notes.